ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -353,7 +362,7 @@ def handleAppletStateUpdateRequested(self):
         batch_processing_busy = self.batchProcessingApplet.busy
 
         self._shell.setAppletEnabled( self.dataSelectionApplet,             not batch_processing_busy )
-        self._shell.setAppletEnabled( self.wsdtApplet,                      not batch_processing_busy and input_ready and not superpixels_available_from_file )
+        self._shell.setAppletEnabled( self.wsdtApplet,                      not batch_processing_busy and input_ready and opDataSelection.ImageGroup[0][1].ready() and not superpixels_available_from_file )
```

Comment:
i don't think the `opDataSelection.ImageGroup[0][1].ready()` is necessary, please remove this

Reply:
i already tried to run without this but its not working . 
so should add this condition.   opDataSelection.ImageGroup[0][1].ready()

Reply:
do you think that maybe the `nRoles` which you have set to `1` could have something to do with it?

Reply:
yes i tried to change nRoles value from 1 to 2 its work without this condition. opDataSelection.ImageGroup[0][1].ready() 
but due to this batch processing also activate only when we add probabilities.
so is it ok with that ?

Reply:
yes, that is okay. Nothing to batch process without a trained project anyway.